### PR TITLE
New version: Microsoft.OneDrive.Enterprise version 22.077.0410.0007

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
### Result: Installation Successful
|             | Manifest                 | Add/Remove Programs                   |
| ----------- | ------------------------ | ------------------------------------- |
| Name        | Microsoft OneDrive Enterprise | Microsoft OneDrive    |
| Version     | 22.077.0410.0007     | 22.077.0410.0007 |
| Publisher   | Microsoft Corporation   | Microsoft Corporation      |
| ProductCode |           | OneDriveSetup.exe    |
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/65765)

### CODE REVIEW

File: manifests/k/Klocman/BulkCrapUninstaller/5.3.0.0/Klocman.BulkCrapUninstaller.installer.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-3
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-5
```
Comment (jedieaston) [inline manifests/k/Klocman/BulkCrapUninstaller/5.3.0.0/Klocman.BulkCrapUninstaller.installer.yaml:1]: :(

**Review by jedieaston — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by vedantmgoyal2009** [comment]
@denelon @hackean-msft @JohnMcPMS @KevinLaMS @msftrubengu @ranm-msft @ryfu-msft @yao-msft @zachcarp

**Review by ItzLevvie — APPROVED** [review]

**Comment by yao-msft** [comment]
2022-07-22 12:16:29.965 [CORE] WinINet downloading from url: https://oneclient.sfx.ms/Win/Enterprise/22.077.0410.0007/amd64/OneDriveSetup.exe
2022-07-22 12:16:32.020 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Downloader.cpp(47)\WindowsPackageManager.dll!00007FFD01B4B989: (caller: 00007FFD01B4C5B4) Exception(4) tid(1cdc) 80072EFD Msg:[InternetOpenUrl() failed.]

**Pull request merged**